api/button.go

```diff
@@ -12,7 +12,7 @@ const (
 	ButtonStyleLink
 )
 
-// NewButton creates a new Button with the provided parameters. Link Button(s) need a url and other Button(s) need a customID
+// NewButton creates a new Button with the provided parameters. Link Button(s) need an url and other Button(s) need a customID
```

Comment:
This is actually incorrect English as URL is an acronym, not a regular word. (it should be capitalized though)